packages/framer-motion/src/gestures/drag/VisualElementDragControls.ts

```diff
@@ -490,7 +490,7 @@ export class VisualElementDragControls {
          * Update the layout of this element and resolve the latest drag constraints
          */
         const { transformTemplate } = this.visualElement.getProps()
-        this.visualElement.getInstance().style.transform = transformTemplate
+        this.visualElement.current!.style.transform = transformTemplate
```

Comment:
This also doesn't look good. There surely must be something we can do with the types here.

Reply:
Yeah we can do a check here for `current` but we're making explicit here something that was actually theorhetically not true (that `getInstance()` would always return an element) even when it effectively was true

Reply:
Ok updated with checks

Reply:
I think if we think it should crash, we should make that explicit (with an assert or throwing an error) if it shouldn't crash, we should just do a check.